src/secq256k1/curve.rs

```diff
@@ -0,0 +1,113 @@
+use crate::ff::WithSmallOrderMulGroup;
+use crate::ff::{Field, PrimeField};
+use crate::group::Curve;
+use crate::group::{prime::PrimeCurveAffine, Group, GroupEncoding};
+use crate::hash_to_curve::svdw_hash_to_curve;
+use crate::secp256k1::{Fp, Fq};
+
+use crate::{
+    batch_add, impl_add_binop_specify_output, impl_binops_additive,
+    impl_binops_additive_specify_output, impl_binops_multiplicative,
+    impl_binops_multiplicative_mixed, impl_sub_binop_specify_output, new_curve_impl,
+};
+use crate::{Coordinates, CurveAffine, CurveAffineExt, CurveExt};
+use core::cmp;
+use core::fmt::Debug;
+use core::iter::Sum;
+use core::ops::{Add, Mul, Neg, Sub};
+use rand::RngCore;
+use subtle::{Choice, ConditionallySelectable, ConstantTimeEq, CtOption};
+
+#[cfg(feature = "derive_serde")]
+use serde::{Deserialize, Serialize};
+
+const SECQ_GENERATOR_X: Fq = Fq::from_raw([
+    0xA24288E37702EDA6,
+    0x3134E45A097781A6,
+    0xB6B06C87A2CE32E2,
+    0x76C39F5585CB160E,
+]);
+
+const SECQ_GENERATOR_Y: Fq = Fq::from_raw([
+    0xA4120DDAD952677F,
+    0xD18983D26E8DC055,
+    0xDC2D265A8E82A7F7,
+    0x3FFC646C7B2918B5,
+]);
+
+const SECQ_A: Fq = Fq::from_raw([0, 0, 0, 0]);
+const SECQ_B: Fq = Fq::from_raw([7, 0, 0, 0]);
+
+new_curve_impl!(
+    (pub),
+    Secq256k1,
+    Secq256k1Affine,
+    true,
+    Fq,
+    Fp,
+    (SECQ_GENERATOR_X, SECQ_GENERATOR_Y),
+    SECQ_A,
+    SECQ_B,
+    "secq256k1",
+    |curve_id, domain_prefix| svdw_hash_to_curve(curve_id, domain_prefix, Secq256k1::SVDW_Z),
+);
+
+impl CurveAffineExt for Secq256k1Affine {
+    batch_add!();
+
+    fn into_coordinates(self) -> (Self::Base, Self::Base) {
+        (self.x, self.y)
+    }
+}
```

Comment:
This has just been removed. So we should rebase and remove the implementation of `CurveAffineExt`

Reply:
yup, [done](https://github.com/privacy-scaling-explorations/halo2curves/pull/65/commits/b0e548f39e25cc4e8ca27c177ecde621e873e1a2)!